<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VM Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconImage.qrc">
    <normaloff>:/icon/images/system.png</normaloff>:/icon/images/system.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>VM</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>cpu</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_VM">
    <property name="title">
     <string>&amp;VM</string>
    </property>
    <addaction name="action_New_VM"/>
    <addaction name="action_Delete_Vm"/>
    <addaction name="separator"/>
    <addaction name="action_Start"/>
    <addaction name="action_Suspend"/>
    <addaction name="action_Shutdown"/>
    <addaction name="action_Kill"/>
    <addaction name="separator"/>
    <addaction name="action_Setting"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_System"/>
   <addaction name="menu_VM"/>
   <addaction name="menu_View"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New_VM"/>
   <addaction name="action_Delete_Vm"/>
   <addaction name="separator"/>
   <addaction name="action_Start"/>
   <addaction name="action_Suspend"/>
   <addaction name="action_Shutdown"/>
   <addaction name="action_Kill"/>
   <addaction name="separator"/>
   <addaction name="action_Setting"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New_VM">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/new.png</normaloff>:/icon/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New VM</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/exit.png</normaloff>:/icon/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/start.png</normaloff>:/icon/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="action_Delete_Vm">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/delete.png</normaloff>:/icon/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete VM</string>
   </property>
  </action>
  <action name="action_Suspend">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/suspend.png</normaloff>:/icon/images/suspend.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Suspend</string>
   </property>
  </action>
  <action name="action_Shutdown">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/shutdown.png</normaloff>:/icon/images/shutdown.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Shutdown</string>
   </property>
  </action>
  <action name="action_Kill">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/kill.png</normaloff>:/icon/images/kill.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Kill</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="iconImage.qrc">
     <normaloff>:/icon/images/setting.png</normaloff>:/icon/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconImage.qrc"/>
 </resources>
 <connections/>
</ui>
